pkg/virt-launcher/virtwrap/converter/converter_test.go

```diff
@@ -2029,6 +2030,74 @@ var _ = Describe("Converter", func() {
 		})
 	})
 
+	Context("Correctly handle IsolateEmulatorThread with dedicated cpus", func() {
+		DescribeTable("should fail assigning CPUs to emulatorThread", func(
+			cores uint32, converterContext *ConverterContext, expectedErrorString string) {
+			var err error
+			domain := &api.Domain{}
+
+			cpuPool := vcpu.NewRelaxedCPUPool(
+				&api.CPUTopology{Sockets: 1, Cores: cores, Threads: 1},
+				converterContext.Topology,
+				converterContext.CPUSet,
+			)
+			domain.Spec.CPUTune, err = cpuPool.FitCores()
+			Expect(err).ToNot(HaveOccurred())
+
+			_, err = vcpu.FormatEmulatorThreadPin(cpuPool)
+			Expect(err).To(MatchError(ContainSubstring(expectedErrorString)))
+		},
+			Entry("when there are not enough CPUs to allocate emulator threads",
+				uint32(2), &ConverterContext{CPUSet: []int{5, 6},
+					AllowEmulation: true,
+					SMBios:         &cmdv1.SMBios{},
```

Comment:
Are there values necessary for the test?

Reply:
looks like they are not. removing.
DONE